doc_src/faq.hdr

```diff
@@ -220,6 +221,11 @@ rm -f fish fish_indent
 
 Fish reserves the <a href="http://www.unicode.org/faq/private_use.html">Unicode private-use character range</a> from U+F600 thru U+F73F for internal use. Any attempt to feed characters in that range to fish will result in them being replaced by the Unicode "replacement character" U+FFFD. This includes both interactive input as well as any file read by fish (but not programs run by fish).
 
+<hr>
+\section faq-third-party Where can I find 3rd-party extensions?
```

Comment:
Although we know them as third-party extensions, I don't think this question means much to new users. What about "Where can I find extra tools for fish?"